core/src/main/scala/com/pingcap/tispark/TiConfigConst.scala

```diff
@@ -28,6 +28,7 @@ object TiConfigConst {
   val REQUEST_ISOLATION_LEVEL: String = "spark.tispark.request.isolation.level"
   val ALLOW_INDEX_READ: String = "spark.tispark.plan.allow_index_read"
   val COPROCESS_STREAMING: String = "spark.tispark.coprocess.streaming"
+  val ENABLE_CHUNK: String = "spark.tispark.coprocess.arrow"
```

Comment:
Please be consistent.

Besides, we should use name "columnar chunk" rather than "chunk" throughout the whole project.